reporters/kamon-prometheus/src/main/resources/reference.conf

```diff
@@ -58,6 +58,26 @@ kamon.prometheus {
     }
   }
 
+  summary: {
```

Comment:
Maybe call it `summaries` instead? Just because the previous one is `buckets`, in plural.

Reply:
done